test/elm/clinical/clinical-test.ts

```diff
@@ -157,6 +161,48 @@ describe('InValueSet', () => {
   });
 });
 
+describe('ExpandValueset', () => {
+  beforeEach(function () {
+    setup(this, data, [], vsets);
+  });
+
+  it('expand valueset', async function () {
+    const received: any = await this.expandFemale.exec(this.ctx);
+    const expected: Code[] = [new Code('F', '2.16.840.1.113883.18.2', 'HL7V2.5', undefined)];
+    (received.length == expected.length).should.true();
+    (received[0] != null).should.true();
+    (received[0].code === expected[0].code).should.true();
+    (received[0].system === expected[0].system).should.true();
+    (received[0].version === expected[0].version).should.true();
+    (received[0].display === expected[0].display).should.true();
```

Comment:
This can be simplified to:
```ts
received.should.eql(expected);
```
This didn't work initially because it was importing `Code` from the `lib` folder instead of the `src` folder. But once you fix that import, `eql` works as expected here.